source/renderer/app/components/status/NetworkStatus.js

```diff
@@ -248,202 +486,292 @@ export default class NetworkStatus extends Component<Props, State> {
+              <tr>
+                <td>{intl.formatMessage(messages.daedalusVersion)}:</td>
+                <td title={daedalusVersion}>{daedalusVersion}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.daedalusMainProcessID)}:</td>
+                <td title={daedalusMainProcessID}>{daedalusMainProcessID}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.daedalusProcessID)}:</td>
+                <td title={daedalusProcessID}>{daedalusProcessID}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.safeMode)}:</td>
+                <td className={styles.safeMode}>
+                  {isInSafeMode ? 'YES' : 'NO'}
                 </td>
               </tr>
               <tr>
-                <td>hasBeenConnected:</td>
-                <td>{hasBeenConnected ? 'YES' : 'NO'}</td>
+                <td>{intl.formatMessage(messages.cardanoVersion)}:</td>
+                <td title={cardanoVersion}>{cardanoVersion}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.cardanoProcessID)}:</td>
+                <td title={cardanoProcessID}>{cardanoProcessID}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.cardanoApiPort)}:</td>
+                <td title={cardanoAPIPort}>{cardanoAPIPort}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.cardanoNetwork)}:</td>
+                <td title={cardanoNetwork}>{cardanoNetwork}</td>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.stateDirectory)}:</td>
+                <td
+                  className={styles.stateDirectory}
+                  title={daedalusStateDirectory}
+                >
+                  {daedalusStateDirectory}
+                </td>
+              </tr>
+              {!isConnected && nodeConnectionError ? (
+                <tr>
+                  <td className={styles.topPadding} colSpan={2}>
+                    {intl.formatMessage(messages.connectionError)}
+                    <br />
+                    <div className={styles.error} title={message || '-'}>
+                      message: {message || '-'}
+                      <br />
+                      code: {code || '-'}
+                    </div>
+                  </td>
+                </tr>
+              ) : null}
+            </tbody>
+          </table>
+
+          <table className={styles.table}>
+            <tbody>
+              <tr>
+                <th colSpan={2}>
+                  {intl.formatMessage(messages.daedalusStatus)}
+                  <hr />
+                </th>
+              </tr>
+              <tr>
+                <td>{intl.formatMessage(messages.connected)}:</td>
+                <td className={this.getClass(isConnected)}>
+                  {isConnected ? 'YES' : 'NO'}
```

Comment:
@DeeJayElly please add a translation for all `YES` and `NO` values 🙏

Reply:
@nikolaglumac done